code/modules/mob/living/silicon/robot/robot_modules/station/drone.dm

```diff
@@ -25,6 +25,9 @@
 	src.modules += new /obj/item/floor_painter(src)
 	src.modules += new /obj/item/t_scanner(src)
 	src.modules += new /obj/item/analyzer(src)
+	src.modules += new /obj/item/inflatable_dispenser/robot(src)
```

Comment:
We are not making maintdrones basicly equivalent to engineering borgs

Reply:
Per discussion on discord, leaving in the new modules